torch/distributions/bernoulli.py

```diff
@@ -77,10 +77,10 @@ def log_prob(self, value):
         if self._validate_args:
             self._validate_sample(value)
         logits, value = broadcast_all(self.logits, value)
-        return -binary_cross_entropy_with_logits(logits, value, reduce=False)
+        return -binary_cross_entropy_with_logits(logits, value, reduction='none')
```

Comment:
These are taken care of in #9162 .

Reply:
Thanks @vishwakftw , I'm gonna revert the last commit once #9162 is merged.